### ISSUE TITLE
move operator target generation to prometheus-support

### ISSUE THREAD

**Issue by stephen-soltesz** [m-lab/prometheus-support#501]
The generation and deployment of targets that used to come from the m-lab/operator repo can be moved to prometheus-support now -- https://github.com/m-lab/operator/blob/master/.travis.yml#L47

**Comment by nkinkade** [comment]
Looking at this more closely, it may not be trivial to move Prometheus target generation from the operator repository to this one. The script `generate_prometheus_targets.sh` still relies heavily on `legacyconfig.py`, which itself relies on `sites.py`.

**Comment by stephen-soltesz** [comment]
Oooh. That's so we can continue to support scraper? We may be tied to retiring scraper then before moving it easily.

**Comment by nkinkade** [comment]
Yeah, to support the scraper "group", which is rsync stuff, but also stuff not directly related to the scraper like ssh806 and sidestream monitoring. Oh, well, I guess this will have to be deferred for now, unless we wanted to split legacy target generation from the rest, which probably wouldn't be a ton of work. What do you think?

**Comment by stephen-soltesz** [comment]
If all the scraper targets will be obsolete on the new platform, then doing the split now could work. prometheus-support generates the global targets, and operator generates the scraper targets?

**Comment by nkinkade** [comment]
@stephen-soltesz and I worked through some of this on VC, and the decision was to split the legacyconfig.py things from the mlabconfig.py things, and to leave the legacyconfig.py things in the operator repo and put the mlabconfig.py things in the prometheus-support repo. 

Additionally, we decided to move mlabconfig.py out of the operator repo and into `m-lab/siteinfo:cmd/mlabconfig.py`. The reasoning behind putting it in a `cmd/` dir is that we'd like to replace the python script with a Go implementation eventually.

As far as I can tell, the only other repository that currently relies on mlabconfig.py is epoxy-images, which clones the operator repo in some scripts. We will simply modify epoxy-images to clone the siteinfo repo instead.
